pkg/minikube/service/service.go

```diff
@@ -275,7 +275,7 @@ func WaitAndMaybeOpenService(api libmachine.API, namespace string, service strin
 	chkSVC := func() error { return CheckService(namespace, service) }
 
 	if err := retry.Expo(chkSVC, time.Duration(interval)*time.Second, time.Duration(wait)*time.Second); err != nil {
-		return errors.Wrapf(err, "Could not find finalized endpoint being pointed to by %s", service)
+		return errors.Wrapf(err, "Service %s was not found in default namespace , please try with 'minikube service %s -n Y'", service, service)
```

Comment:
I love this text, but it should detect if `namespace` is default, otherwise the error message doesn't make much sense.

Reply:
Made changes to the string to include the service name.

Reply:
@tstromberg any feedback ? Thanks